<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeedbackDlg</class>
 <widget class="QDialog" name="FeedbackDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FeedbackDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#wgt_main{
background:#262626;
border:1px solid #585858;
}
QLabel,QPushButton,QTextEdit{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#999999;
background:transparent;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wgt_main" native="true">
     <property name="styleSheet">
      <string notr="true">#wgt_title{
background:#2F2F2F;
}
#button_commit{
background-image: url(:/TQLive/images/btn_confirm.png);
background-repeat:no-repeat;
border:none;
}
#button_commit:hover{
background-image: url(:/TQLive/images/btn_confirm_hover.png);
}
#button_commit:pressed{
margin:1px;
padding:1px;
}
#label{
font-size:14px;
color:#4AB134;
}
#label_tips{
font-size:14px;
color:#DCDCDC;
}
#textEdit_feedback{
border:1px solid #585858;
border-radius:2px;
color:#DCDCDC;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="wgt_title" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>34</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#label_title{
font-size:14px;
color:#4AB134;
}
#button_close{
background-image: url(:/TQLive/images/pb_close_pressed.png);
background-repeat:no-repeat;
border:none;
}
#button_close:hover{
background-image: url(:/TQLive/images/pb_close_normal.png);
}
#button_close:pressed{
margin:1px;
padding:1px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>19</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_title">
           <property name="text">
            <string>意见反馈</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_close">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>13</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>13</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>22</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>*</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_tips">
            <property name="text">
             <string>请具体描述您遇到的问题</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_feedback">
          <property name="placeholderText">
           <string>描述字数不得超过200个字</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>22</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_commit">
            <property name="minimumSize">
             <size>
              <width>82</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>82</width>
              <height>26</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TQLive.qrc"/>
 </resources>
 <connections/>
</ui>
